h2o-core/src/test/java/water/fvec/TestFrameBuilder.java

```diff
@@ -0,0 +1,247 @@
+package water.fvec;
+
+import water.DKV;
+import water.Key;
+
+import java.util.HashMap;
+
+/**
+ * Class used for creating simple test frames using builder pattern
+ * <p>
+ * Example usage:
+ * <pre>
+ * {@code
```

Comment:
`<pre>` and `{@code` should be on the same line, otherwise the indentation gets messed up... You can hover on class' name in IntelliJ and press F1 to see the difference